src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,58 @@
+import java.util.LinkedList
+import java.util.Queue
+
+object Calculator {
+
+    fun calEquation(equation: String?): Double {
+        if (equation == null || equation.isEmpty()) {
+            throw IllegalArgumentException("입력 값이 유효하지 않습니다.")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(!text.isNullOrBlank()) { "입력 값이 유효하지 않습니다." }
```

Reply:
if 문을 통한 null check 를 하여 throw 를 발생시키는 경우에는 if 문 아래부터는 null check 를 하지 않아도 되지만
`require` 을 사용하는 경우에는 null check 를 밑에도 추가로 해줘야 하는데 이 부분에 대해서는 어떻게 생각하시나요?

Reply:
`text`는 `String`으로 스마트 캐스트(smart cast) 됩니다.

Reply:
require 문을 잘 못 알고 있었습니다.:sob:
적용하였습니다.
[e1405198d0e51e92a683367bdae8e334eb0dcb94]